File: CarShop_Mobile/app/src/main/java/com/example/carshop_mobile/main/Model/UserMobile.kt
package com.example.carshop_mobile.main.Model

import com.google.gson.annotations.SerializedName


class UserMobile{
    @SerializedName("idUserMobile")
    var iDUser :Int? = null
    @SerializedName("username")
    var username : String? = null
    @SerializedName("email")
    var email : String? = null
    @SerializedName("password")
    var password : String? = null
    @SerializedName("mobileTelephone")
    var mobileTelephone : String? = null
    @SerializedName("name")
    var name : String? = null
    @SerializedName("surname")
    var surname : String? = null

    constructor(username:String,password:String,email:String,mobilePhone:String,name:String,surname:String){
        this.username = username
        this.email = email
        this.password = password
        this.mobileTelephone = mobilePhone
        this.name = name
        this.surname = surname
    }

}

File: CarShop_REST_API/CarShop_REST_API/Model/UserMobile.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace CarShop_REST_API.Model
{
    public class UserMobile
    {
        [Key]
        public int IDUserMobile { get; set; }
        public string Username { get; set; }
        public string Email { get; set; }
        public string Password { get; set; }

    }
}
